src/main/java/org/opentripplanner/routing/edgetype/StreetEdgeTraversal.java

```diff
@@ -208,6 +213,25 @@ public StateEditor doTraverse(RoutingRequest options, TraverseMode traverseMode)
                 // TODO: we don't have this data yet
             }
 
+            OptionAttribute curbRamp = extraOptions.getOption(NihOption.CURB_RAMP);
+            if (curbRamp == CurbRamp.YES) {
+                // prefer audited edges with curb ramps
+                weight *= 0.9;
```

Comment:
OTP does use enums for a couple of constant weights.  Since we have lots of them in a fluid state at the moment, I like that we can see the values directly where they're used for now, for comparison without having to scroll around.  I'll make a point of setting up enums for them when finishing up the "weighting" card (this is not that).